llvm/unittests/Target/AArch64/SMEAttributesTest.cpp

```diff
@@ -50,6 +54,22 @@ TEST(SMEAttributes, Constructors) {
                       ->getFunction("foo"))
                   .preservesZA());
 
+  ASSERT_TRUE(SA(*parseIR("declare void @foo() \"aarch64_sme_zt0_in\"")
```

Comment:
Should we change the names of the attributes from `aarch64_sme_zt0_in` to the `aarch64_sme_in_zt0` format as well?  (and similar below)

Reply:
I've changed the attributes to use this format, I think this makes sense given the Clang attribute will be `__arm_in("zt0")`